shell/platform/darwin/macos/framework/Source/FlutterEngine_Internal.h

```diff
@@ -53,6 +53,7 @@ typedef NS_ENUM(NSInteger, FlutterAppExitResponse) {
 @interface FlutterEngineTerminationHandler : NSObject
 
 @property(nonatomic, readonly) BOOL shouldTerminate;
+@property(nonatomic, readwrite) BOOL bindingIsReady;
```

Comment:
Is there a name that also includes raw `dart:ui` apps? Maybe `systemChannelEnabled`?

```suggestion
@property(nonatomic, readwrite) BOOL systemChannelEnabled;
```

Reply:
Maybe `applicationLifecycleIsEnabled`?